src/server/scripts/Custom/SkipDK.cpp

```diff
@@ -340,6 +155,43 @@ class Trinitycore_optional_deathknight_skip : public CreatureScript
     }
 };
 
+void Trinitycore_skip_deathknight_HandleSkip(Player* player)
+{
+    //Not sure where DKs were supposed to pick this up from, leaving as the one manual add
+    player->AddItem(6948, true); //Hearthstone
```

Comment:
dks are given a hearthstone when they check in with their first inn after completing the final turned of their faction's quest line dealing with the dk starter area per retail.

Reply:
Thought that might be the case, I didn't think it would hurt to leave this in from your implemnentation

Reply:
also another note, just giving them the hearthstone wouldnt be wrong either. As u pretty much would of binded with a innkeeper when u get done with the area anyways.